### PR TITLE
Adding Position Roles

### PR DESCRIPTION
**Pull request by DLance96**
I added some basic roles for exec positions. Changes to roles are expected.

### CODE REVIEW

**Comment by ajm188** [comment]
You should link your GitHub account to whatever email you are committing with.

File: organizational/position_roles.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+**President**
+
+- Runs general body meetings
+- Manages the organization ensuring others accomplish specified tasks
+
+**Vice President of ACM**
+
+- Maintain engagement with `ACM <http://www.acm.org/>`_
+- Keep involvment with CS/CE portion of the department
```
Comment (unforced) [inline organizational/position_roles.rst]: We're forgetting the oft-forgotten fourth major in EECS. Systems and control. Not sure if they fall under ACM or IEEE

File: organizational/position_roles.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
+Position Roles 
+========================
+
+Below are the positions that have allowed Hacker's Society function in the past. The
```
Comment (unforced) [inline organizational/position_roles.rst]: Grammar seems off.
"allowed Hacker Society to function" or "allowed Hacker's Society functioning" or something of the sort

File: organizational/position_roles.rst [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+**Public Relations**
+
+- Create and distrubute information within the campus community
+- Contact companies involving events and their involvement
+- Establish sponorship plans for events
+
+**Secretary**
+
+- Keep minutes for general body meetings
```
Comment (unforced) [inline organizational/position_roles.rst:40]: Perhaps include a bit more here. Organizational things such as keeping up on google drive, mailing lists, or whatever else falls under the purview of secretary.

File: organizational/position_roles.rst [hunk 4]
```diff
@@ -0,0 +1,41 @@
+Position Roles
```
Comment (timtadh) [inline organizational/position_roles.rst]: Change to ACM and IEEE Officer Roles

File: organizational/position_roles.rst (hunk 1 above)
Comment (timtadh) [inline organizational/position_roles.rst]: Of course they fall in neither but that is neither here nor there.

**Comment by mtbentley** [comment]
Question: should it be "Hacker's Society", "Hackers Society", Hackers' Society", or "Hacker Society"?

**Comment by timtadh** [comment]
It is "Hacker Society"

**Comment by brenns10** [comment]
Might as well go in at this point.

**Pull request merged**